.github/workflows/integration-tests.yml

```diff
@@ -63,6 +63,27 @@ jobs:
       job-tag: ${{ inputs.job-tag }}
     secrets: inherit
 
+  arm-integration-tests:
```

Comment:
it might be worth renaming `all-integration-tests` and `required-integration-tests` to something like `amd64-required-tests` and `amd64-additional-tests` to be consistent with this. I think it'll make things a lot clearer, especially with s390x and ppc64le added in for good measure (I'd advocate for renaming the multiarch test job as well, though there'll be more duplicated boilerplate for this, given there's only a single vm_type for each)

Reply:
Makes sense, I'll rename the tests.